vivarium/core/engine.py

```diff
@@ -592,6 +550,22 @@ def _make_store(
             self.flow = self.state.get_flow() or {}
             self.topology = self.state.get_topology()
 
+    def _parallelize_processes(
+            self, processes: Any) -> Union[dict, Process]:
```

Comment:
Just to document my thought process, I decided to swap processes for `ParallelProcess` instances in `Engine` instead of in `Store` because we need to know whether to enable the profiler when creating the `ParallelProcess`. `Engine` has that information, but it felt wrong to pass that information through to all the `Store` nodes.